File: QuickStart/PropertyGrid/PropertyStore/Enumerations.vb

Imports System

Namespace Telerik.Examples.WinControls.PropertyGrid.PropertyStore
    <Flags()> _
    Public Enum EngineLayouts
        I4 = 1
        I6 = 2
        V6 = 4
        V8 = 8
        V10 = 16
        V12 = 32
    End Enum

    <Flags()> _
    Public Enum Fuels
        Petrol = 1
        Diesel = 2
        Ethanol = 4
        Electric = 8
    End Enum

    <Flags()> _
    Public Enum Transmissions
        Manual = 1
        SemiAutomatic = 2
        Automatic = 4
    End Enum

    <Flags()> _
    Public Enum Drives
        FrontWheelDrive = 1
        RearWheelDrive = 2
        AllWheelDrive = 4
    End Enum
End Namespace

File: QuickStart/PropertyGrid/PropertyStore/Enumerations.cs
using System;

namespace Telerik.Examples.WinControls.PropertyGrid.PropertyStore
{
    [Flags]
    public enum EngineLayouts
    {
        I4 = 1,
        I6 = 2, 
        V6 = 4, 
        V8 = 8, 
        V10 = 16, 
        V12 = 32
    }

    [Flags]
    public enum Fuels
    {
        Petrol = 1, 
        Diesel = 2, 
        Ethanol = 4, 
        Electric = 8
    }

    [Flags]
    public enum Transmissions
    {
        Manual = 1, 
        SemiAutomatic = 2, 
        Automatic = 4
    }

    [Flags]
    public enum Drives
    {
        FrontWheelDrive = 1, 
        RearWheelDrive = 2, 
        AllWheelDrive = 4
    }
}
